src/commands/BootstrapCommand.js

```diff
@@ -92,12 +97,32 @@ export default class BootstrapCommand extends Command {
   }
 
   /**
-   * Run the "prepublish" NPM script in all bootstrapped packages
+   * Run the "prepublish" or "prepare" (depending on the version of npm client)
+   * NPM script in all bootstrapped packages
    * @param callback
    */
   prepublishPackages(callback) {
     this.logger.info("Prepublishing packages");
-    this.runScriptInPackages("prepublish", callback);
+
+    const npmClientVersionString = NpmUtilities.getNpmClientVersion();
+    const npmClientMajorVersion = parseInt(npmClientVersionString, 10);
+
+    if (npmClientMajorVersion <= 3) {
```

Comment:
I'm just gonna point out that we don't actually _need_ to check the npm client version here -- `npm run prepare` will work just as well on 2 as on 4.  That being said, I think you made the right call: it's super important that users be able to fall back to `npm` if lerna isn't doing what they expect, and papering over the version mismatch would preclude that.